tmp/biharmonic_coordinates.cpp

```diff
@@ -81,23 +82,25 @@ npe_doc(ds_biharmonic_coordinates)
 
 npe_arg(v, dense_float, dense_double)
 npe_arg(t, dense_float, dense_double)
-npe_arg(s, std::vector<std::vector<SType> > &)
+npe_arg(s, std::vector<std::vector<int> > &)
```

Comment:
Let's add a unit test to make sure this gets properly bound with pybind (try with ints and longs just to check)

Reply:
This part I'm still debugging, it fails to compile, so it is in the tmp directory which will not be compiled.